packages/scorchwrap/src/runtime/runtime.js

```diff
@@ -0,0 +1,159 @@
+// @ts-check
+/// <reference path="./runtime.d.ts" />
+
+const { create, freeze, assign, defineProperty, entries, fromEntries, getOwnPropertyDescriptors } = Object;
+
+// Avoid running any wrapped code or using compartment if lockdown was not called.
+// This is for when the bundle ends up running despite SES being missing.
+// It was previously useful for sub-compilations running an incomplete bundle as part of the build, but currently that is being skipped. We might go back to it for the sake of build time security if it's deemed worthwihile in absence of lockdown.
+const LOCKDOWN_ON = typeof lockdown !== "undefined";
+if (LOCKDOWN_ON) {
+  lockdown(LAVAMOAT.options.lockdown);
```

Comment:
```suggestion
  lockdown(LAVAMOAT.options.lockdown);
  harden(LAVAMOAT);
```
?

Reply:
Yes. A matter of finding the right time for it though.